bin/launch_data_transformer.py

```diff
@@ -38,7 +38,7 @@ def main(data_csv, config_json, out_path):
     
     # Transform the data according to what defined in the experiment class and the specifics of the user in the Json
     # in case of no transformation specification so when the config has "augmentation" : None  just save a copy of the original csv file
-    if config["transform"]: 
+    if "transform" in config: 
```

Comment:
no we want to know if the key associated to tranform is not None, not that the keyword "transform" is in the config. for the moment let's leave it like it is.

```suggestion
    if config["transform"]: 
```